classes/class-twenty-twenty-one-custom-colors.php

```diff
@@ -86,9 +88,16 @@ public function custom_color_variables() {
 	 * Editor custom color variables.
 	 *
 	 * @access public
+	 *
+	 * @return void
 	 */
 	public function editor_custom_color_variables() {
-		wp_enqueue_style( 'twenty-twenty-one-custom-color-overrides', get_template_directory_uri() . '/assets/css/custom-color-overrides.css', array(), wp_get_theme()->get( 'Version' ) );
+		wp_enqueue_style(
+			'twenty-twenty-one-custom-color-overrides',
+			get_theme_file_uri( 'assets/css/custom-color-overrides.css' ),
+			array(),
+			(string) filemtime( get_theme_file_path( 'assets/css/custom-color-overrides.css' ) )
```

Comment:
This is ok for development, but not that ok for production. This would invalidate any caching set in place (`filemtime`).

Reply:
:thinking: Why would it not be OK for production?
Caches will only be invalidated when the modification date of the file changes, which would indicate that the contents of the file have also changed. On a production environment the files don't change, and if they do then caches have to be invalidated (which wouldn't happen with a static version number). Unless I'm missing something?
I'd agree if it was `time()`, that is definitely not suitable for production :sweat_smile:

Reply:
Oh, my bad. It's ok for caching, but I know there was a reason it is discouraged:

https://github.com/WordPress/WordPress-Coding-Standards/pull/1378#issuecomment-398784410

😄